### ISSUE TITLE
Feature Request: Allow manual selection of config.plist at boot time

### ISSUE THREAD

**Issue by TonyC5** [acidanthera/bugtracker#1430]
It would be very helpful if OpenCore would permit manual selection of the active config.plist at boot time.  This feature has been useful with other bootloaders when testing a new config.plist. With this new feature, a stable "baseline" config.plist could remain in the EFI/OC directory.  This stable "baseline" config.plist would be a viable fallback if a new config.plist prevented booting for any reason.

**Comment by vit9696** [comment]
OpenCore configuration cannot be changed once OpenCore is loaded, as the configuration is applied immediately at startup. To change the configuration for OpenCore you must set it before OpenCore starts.

Since 0.6.4 you can load OpenCore from any folder by running `<OC_DIR>\Bootstrap\Bootstrap.efi`, where `<OC_DIR>` can be anything. So by having some kind of boot selector app that runs before OpenCore and lets you choose these OpenCore versions you can achieve exactly what you want.

Such an app is not bundled with OpenCore, so you can either use UEFI Shell or find/write something custom. While I can imagine merging something like that into master branch if it is contributed externally, we have no such plans ourselves.

**Comment by TonyC5** [comment]
@vit9696 I apologize for bothering you with this feature request.  I had started authoring this request while I was researching the possibility and had already concluded that the request was not possible (and that you had been asked for similar capability already).  I had abandoned this request (thinking it wasn't submitted), but it apparently was auto-saved by github.  Again, I apologize for bothering you with this and appreciate that you took the time to respond and explain.  Thank you and thanks for all you're doing with OC - Amazing work.
